docs/faq.md

```diff
@@ -28,11 +32,11 @@ Some metrics like the number of bytes, HTTP headers, etc are accurate at the tim
 The time measurements are gathered in a test environment, and thus have all the potential biases that come with that:
 
 - browser - All tests are performed using a single browser. Page load times can vary depending on browser.
-- location - The HAR files are generated from Redwood City, CA. The distance to the site's servers can affect time measurements.
-- sample size - Each URL is loaded three times. The HAR file is generated from the median test run. This is not a large sample size.
+- location - The HAR files are generated from various datacenters in the US. The distance to the site's servers can affect time measurements.
+- sample size - Each URL is loaded only once.
 - Internet connection - The connection speed, latency, and packet loss from the test location is another variable that affects time measurements.
 
-Given these conditions it's virtually impossible to compare WebPagetest.org's time measurements with those gathered in other browsers or locations or connection speeds. They are best used as a source of comparison.
+Given these conditions it's virtually impossible to compare the HTTP Archive's time measurements with those gathered in other browsers, locations or connection speeds. They are best used as a source of comparison.
```

Comment:
You didn't change it, but the last sentence doesn't really make sense to me. Clarify it?

```suggestion
Given these conditions it's virtually impossible to compare the HTTP Archive's time measurements with those gathered in other browsers, locations or connection speeds. They are best used as a source of comparison only within the HTTP Archive tests.
```